modules/storages/app/services/storages/group_folder_properties_sync_service.rb

```diff
@@ -50,11 +51,15 @@ def initialize(storage)
   # rubocop:disable Metrics/AbcSize
   def call
     # we have to flush state to be able to reuse the instace of the class
-    @nextcloud_usernames_used_in_openproject = Set.new
     @project_folder_ids_used_in_openproject = Set.new
     @file_ids = nil
     @folders_properties = nil
     @group_users = nil
+    @admin_tokens_query = OAuthClientToken.where(oauth_client: @storage.oauth_client,
+                                                 users: User.where(admin: true, status: 'active'))
```

Comment:
```suggestion
    @admin_tokens_query = OAuthClientToken.where(oauth_client: @storage.oauth_client,
                                                 users: User.admin.active)
```

Using scopes might be more complete here. Apparently, the `User.active` scope ensures we exclude  system users, deleted users and anonymous users- via `Principals::Scopes::Status`. Is this behaviour valid for this use case?

```ruby
[7] pry(main)> User.admin.active.to_sql
=> "SELECT \"users\".* FROM \"users\" WHERE \"users\".\"type\" IN ('User', 'DeletedUser', 'AnonymousUser', 'SystemUser') AND \"users\".\"admin\" = TRUE AND \"users\".\"type\" NOT IN ('SystemUser', 'AnonymousUser', 'DeletedUser') AND \"users\".\"status\" = 1"
[8] pry(main)> User.where(admin: true, status: 'active').to_sql
=> "SELECT \"users\".* FROM \"users\" WHERE \"users\".\"type\" IN ('User', 'DeletedUser', 'AnonymousUser', 'SystemUser') AND \"users\".\"admin\" = TRUE AND \"users\".\"status\" = 1"
```

Reply:
Yes, it makes sense for sure. Thanks!